lib/krikri/engine.rb

```diff
@@ -75,6 +75,11 @@ def configure_blacklight!
     initializer :rdf_repository do
       Krikri::Repository =
         RDF::Marmotta.new(Krikri::Settings['marmotta']['base'])
+
+      Krikri::Repository.query_client.instance_eval do
+        @http.keep_alive = 360
```

Comment:
Have you considered using Figaro? Or dotenv? This would be to set an application configuration variable.

https://www.ruby-toolbox.com/categories/Configuration_Management